### PR TITLE
[NO_TICKET] Upgrade to CMSDS@v2.2.1 

### PR DESCRIPTION
**Pull request by nichia**
### Summary
- Upgrade to CMSDS@v2.2.1 
- Update the order of SCSS imports to be consistent with core design system.
- Adds error icon to error messages

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
 
 // Components: UI component styles (i.e. ds-c-button, ds-c-field)
 @import "@cmsgov/design-system/dist/scss/components/index";
-@import "components/index";
\ No newline at end of file
+@import "components/index";
+
+// Utilities: Utility classes for applying specific CSS properties
+@import "@cmsgov/design-system/dist/scss/utilities/index";
+//@import "utilities/index";
```
Comment (bernardwang) [inline src/styles/index.scss:26]: Needs new line here

**Review by bernardwang — APPROVED** [review]
LGTM

File: src/styles/index.scss (hunk 1 above)
Comment (nichia) [reply src/styles/index.scss:26]: It's just a commented placeholder.

Comment (bernardwang) [reply src/styles/index.scss:26]: No I mean every file should end with a new line. Its a linting rule

**Review by line47 — APPROVED** [review]

**Comment by line47** [comment]
I think this can be merged @nichia

**Pull request merged**